---
title: "Re-evaluating the role of nucleosomal bivalency in early development"

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors:
- admin
- Adrian Grzybowski
- Jimmy Elias
- Zhonglei Chen
- Takamitsu Hattori
- Carolin Lechner
- Peter Lewis
- Shohei Koide
- Beat Fierz
- Alexander Ruthenburg

# Author notes (optional)
author_notes:
- Equal Contribution
- Equal Contribution
- Equal Contribution

date: "2021-09-10"
doi: "10.1101/2021.09.09.458948"

# Schedule page publish date (NOT publication's date).
publishDate: "2021-09-10T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["3"]

# Publication name and optional abbreviated publication name.
publication: In *bioRxiv*
publication_short: In *bioRxiv*

abstract: "Nucleosomes, composed of DNA and histone proteins, represent the fundamental repeating unit of the eukaryotic genome; posttranslational modifications of these histone proteins influence the activity of the associated genomic regions to regulate cell identity. Traditionally, trimethylation of histone 3-lysine 4 (H3K4me3) is associated with transcriptional initiation, whereas trimethylation of H3K27 (H3K27me3) is considered transcriptionally repressive. The apparent juxtaposition of these opposing marks, termed “bivalent domains”, was proposed to specifically demarcate a small set of transcriptionally-poised lineage-commitment genes that resolve to one constituent modification through differentiation, thereby determining transcriptional status. Since then, many thousands of studies have canonized the bivalency model as a chromatin hallmark of development in many cell types. However, these conclusions are largely based on chromatin immunoprecipitations (ChIP) with significant methodological problems hampering their interpretation. Absent direct quantitative measurements, it has been difficult to evaluate the strength of the bivalency model. Here, we present reICeChIP, a calibrated sequential ChIP method to quantitatively measure H3K4me3/H3K27me3 bivalency genome-wide, addressing the limitations of prior measurements. With reICeChIP, we profile bivalency through the differentiation paradigm that first established this model: from naïve mouse embryonic stem cells (mESCs) into neuronal progenitor cells (NPCs). Our results cast doubt on every aspect of the bivalency model; in this context, we find that bivalency is widespread, does not resolve with differentiation, and is neither sensitive nor specific for identifying poised developmental genes or gene expression status more broadly. Our findings caution against interpreting bivalent domains as specific markers of developmentally poised genes."

# Summary. An optional shortened abstract.
summary: "Nucleosomes, composed of DNA and histone proteins, represent the fundamental repeating unit of the eukaryotic genome; posttranslational modifications of these histone proteins influence the activity of the associated genomic regions to regulate cell identity. Traditionally, trimethylation of histone 3-lysine 4 (H3K4me3) is associated with transcriptional initiation, whereas trimethylation of H3K27 (H3K27me3) is considered transcriptionally repressive. The apparent juxtaposition of these opposing marks, termed “bivalent domains”, was proposed to specifically demarcate a small set of transcriptionally-poised lineage-commitment genes that resolve to one constituent modification through differentiation, thereby determining transcriptional status. Since then, many thousands of studies have canonized the bivalency model as a chromatin hallmark of development in many cell types. However, these conclusions are largely based on chromatin immunoprecipitations (ChIP) with significant methodological problems hampering their interpretation. Absent direct quantitative measurements, it has been difficult to evaluate the strength of the bivalency model. Here, we present reICeChIP, a calibrated sequential ChIP method to quantitatively measure H3K4me3/H3K27me3 bivalency genome-wide, addressing the limitations of prior measurements. With reICeChIP, we profile bivalency through the differentiation paradigm that first established this model: from naïve mouse embryonic stem cells (mESCs) into neuronal progenitor cells (NPCs). Our results cast doubt on every aspect of the bivalency model; in this context, we find that bivalency is widespread, does not resolve with differentiation, and is neither sensitive nor specific for identifying poised developmental genes or gene expression status more broadly. Our findings caution against interpreting bivalent domains as specific markers of developmentally poised genes."

# Display this page in the Featured widget?
featured: true
---
