---
title: "Examining the Roles of H3K4 Methylation States with Systematically Characterized Antibodies"

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors:
- Adrian Grzybowski
- admin
- William Richter
- Alexander Ruthenburg

# Author notes (optional)
author_notes:
- Equal Contribution
- Equal Contribution


date: "2019-11-13"
doi: "10.1038/s41596-019-0218-7"

# Schedule page publish date (NOT publication's date).
publishDate: "2019-11-13T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: In *Nature Protocols*
publication_short: In *Nat Protoc*

abstract: Chromatin immunoprecipitation coupled to next-generation sequencing (ChIP-seq) has served as the central method for the study of histone modifications for the past decade. In ChIP-seq analyses, antibodies selectively capture nucleosomes bearing a modification of interest and the associated DNA is then mapped to the genome to determine the distribution of the mark. This approach has several important drawbacks: (i) ChIP interpretation necessitates the assumption of perfect antibody specificity, despite growing evidence that this is often not the case. (ii) Common methods for evaluating antibody specificity in other formats have little or no bearing on specificity within a ChIP experiment. (iii) Uncalibrated ChIP is reported as relative enrichment, which is biologically meaningless outside the experimental reference frame defined by a discrete immunoprecipitation (IP), thus preventing facile comparison across experimental conditions or modifications. (iv) Differential library amplification and loading onto next-generation sequencers, as well as computational normalization, can further compromise quantitative relationships that may exist between samples. Consequently, the researcher is presented with a series of potential pitfalls and is blind to nearly all of them. Here we provide a detailed protocol for internally calibrated ChIP (ICeChIP), a method we recently developed to resolve these problems by spike-in of defined nucleosomal standards within a ChIP procedure. This protocol is optimized for specificity and quantitative power, allowing for measurement of antibody specificity and absolute measurement of histone modification density (HMD) at genomic loci on a biologically meaningful scale enabling unambiguous comparisons. We provide guidance on optimal conditions for next-generation sequencing (NGS) and instructions for data analysis. This protocol takes between 17 and 18 h, excluding time for sequencing or bioinformatic analysis. The ICeChIP procedure enables accurate measurement of histone post-translational modifications (PTMs) genome-wide in mammalian cells as well as Drosophila melanogaster and Caenorhabditis elegans, indicating suitability for use in eukaryotic cells more broadly.

# Summary. An optional shortened abstract.
summary: Chromatin immunoprecipitation coupled to next-generation sequencing (ChIP-seq) has served as the central method for the study of histone modifications for the past decade. Here we provide a detailed protocol for internally calibrated ChIP (ICeChIP), a method we recently developed to resolve these problems by spike-in of defined nucleosomal standards within a ChIP procedure. This protocol is optimized for specificity and quantitative power, allowing for measurement of antibody specificity and absolute measurement of histone modification density (HMD) at genomic loci on a biologically meaningful scale enabling unambiguous comparisons. We provide guidance on optimal conditions for next-generation sequencing (NGS) and instructions for data analysis. This protocol takes between 17 and 18 h, excluding time for sequencing or bioinformatic analysis. The ICeChIP procedure enables accurate measurement of histone post-translational modifications (PTMs) genome-wide in mammalian cells as well as Drosophila melanogaster and Caenorhabditis elegans, indicating suitability for use in eukaryotic cells more broadly.

# Display this page in the Featured widget?
featured: true
---
